test/fixtures/hello-cloud/template-runner.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: PodTemplate
+metadata:
+  name: hello-cloud-template-runner
+  labels:
+    name: hello-cloud-template-runner
+    app: hello-cloud
+template:
+  metadata:
+    name: task-runner
+    labels:
+      name: runner
+      app: hello-cloud
+  spec:
+    containers:
+    - name: task-runner
+      image: hello-world:latest
```

Comment:
This means it'll run the latest production build, not the latest deployed one. Perhaps we should convert this into a template and allow running an arbitrary SHA and default to `latest`, or something like this..